app/services/session_store.rb

```diff
@@ -0,0 +1,44 @@
+# Provides a wrapper for accessing the redis cache out-of-band (using only the
+# session UUID) instead of having access to the user session from Devise/Warden.
+# Should only used outside of a normal browser session (such as the OpenID
+# Connect API or remote SAML Logout).
+
+class SessionStore
```

Comment:
WDYT of renaming to `OutOfBandSession` or `OutOfBandSessionAccessor` or something?

Since we need the `::SessionStore` in `Pii::SessionStore` and the internal object is called `session_store` ... that's a lot of the same name over and over again, makes it hard to tell what does what

Reply:
yeah I wasn't thrilled with it but mostly just copied the old code around. Happy to update.